packages/react-components/react-alert/package.json

```diff
@@ -19,6 +19,7 @@
     "just": "just-scripts",
     "lint": "just-scripts lint",
     "start": "yarn storybook",
+    "start-test": "yarn test --watch",
```

Comment:
```suggestion
```

Let's avoid adding new scripts

Reply:
@khmakoto suggested we add this since there's no way to run UTs in watch mode.

Reply:
/cc @Hotell

Reply:
>  suggested we add this since there's no way to run UTs in watch mode.

that's incorrect information, you can use all valid jest CLI arguments via `test` alias as it runs raw `jest`. In past v8 used to have this kind of setup because of limitations with just-scripts. Please remove this alias. ty

example of running jest in watch mode:

`yarn workspace @fluentui/react-alert test --watch`

Reply:
Yeah, I said you could use `test --watch` or add a script to run that, I agree with the idea of not adding this new script.

Reply:
Removed!